src/main/java/io/jenkins/plugins/checks/github/GitHubChecksPublisher.java

```diff
@@ -35,6 +36,9 @@ public class GitHubChecksPublisher extends ChecksPublisher {
     private final PluginLogger buildLogger;
     private final String gitHubUrl;
 
+    @Nullable
+    private StandardUsernameCredentials credentials;
```

Comment:
`credentials` where moved from method variable to class field. This helps `GitHubChecksPublisherITest` to check that owner is propagated from context to credentials.